src/App/App.js

```diff
@@ -59,6 +59,18 @@ class App extends Component {
     });
   }
 
+  getExpandedDeckInfo = (obj) => {
+    let deck = this.state.cards.reduce((acc, currentCardObj) => {
+      obj.cards.forEach(card => {
+        if (card === currentCardObj.cardName) {
+          acc.push(currentCardObj);
+        }
+      })
+      return acc;
+    }, [])
+    return deck;
+  }
```

Comment:
obj is a deck, right? If i'm understanding this correctly, this method returns an array of cards that match the card names in obj.cards. Could this be done with a filter? I'm thinking something like this:
```
getExpandedDeckInfo = (deck) => {
  return this.state.cards.filter(card => {
    return deck.cards.includes(card.cardName);
  });
}
```

Reply:
Sweet. I updated it. Your suggestion is much easier to read!